src/renderers/html.test.ts

```diff
@@ -1,39 +1,43 @@
 import render from './html';
 import { RenderableTreeNode } from '../types';
+import Tag from '../tag';
 
 function tag(
   name: string,
   attributes: Record<string, any> = {},
   children: RenderableTreeNode[] = []
 ) {
-  return { name, attributes, children };
+  return new Tag(name, attributes, children);
 }
 
 describe('HTML renderer', function () {
   it('rendering a tag', function () {
-    const example = render(tag('h1', null, ['test'])).trim();
+    const example = render(tag('h1', undefined, ['test'])).trim();
```

Comment:
is intentional to have the `tag` helper function instead of using `Tag` class directly in the tests? (I was initially confused because I missed the function and was confused why we were passing `undefined`

Reply:
I think it's just to make the test cases smaller. We could replace with `new Tag` if we wanted.